templates/download/core/intel-nuc.html

```diff
@@ -0,0 +1,213 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on the Intel NUC{% endblock %}}
+
+{% block content %}
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-10">
+      <div>
+        <h1>Install Ubuntu on the Intel NUC</h1>
+        <p>There are two install options for the Intel NUC: Ubuntu Core or Ubuntu Desktop.</p>
```

Comment:
question... do you think these should be internal links?

Reply:
Yeah I think that's a good idea